pages/03-01 primitives-referenced-values.md

```diff
@@ -0,0 +1,264 @@
+# Primitives, composite data types  and referenced values
+
+
+## Introduction
+
+```lua
+    local string1 = "hello"
+    local string2 = 'hello'
+
+    print(string1 == string2)
+
+    local number1 = 14
+    local number2 = 14
+
+    print(number1 == number2)
+
+    local table1 = {}
+    local table2 = {}
+
+    print(table1 == table2)
+
+    local function1 = function() end
+    local function2 = function() end
+
+    print(function1 == function2)
+
+    What would happen if you were to run this?
+    In chapter 1 we learned about comparing string with the == operator when we talked about booleans.
+    Let's run the code in the repl and try it out:
+
+    true
+    true
+    false
+    false
+
+    The strings equal and the numbers equal, but why aren't the tables and functions equal since they are both empty?
+```
+
+First, let's see variables types and their meanings.
+	
+	
+### Primitves data types
+In Lua, though we don't have variable data types, but we have types for the values. The list of data types for values are given below.
```

Comment:
Lua's data types is covered in the first chapter so listing the data types here won't be necessary. The focus should be on what the difference is between primitives and composites from a language-agnostic point of view.